libstuff/STCPNode.h

```diff
@@ -101,6 +101,9 @@ struct STCPNode : public STCPManager {
         atomic<Response> transactionResponse;
         atomic<string> version;
 
+        // An address on which this peer can accept commands.
+        atomic<string> commandAddress;
```

Comment:
Why do we need this? Since this is set to the control port elsewhere, couldn't we just use the control port?

Reply:
This is the address of the place we can send commands to on the peer. We don't have this information otherwise unless the peer sends it to us.

Reply:
I mean later you do `_syncNode->setCommandAddress(args["-controlPort"]);` so we commandAddress to the value already set in `controlPort`. Why not just use controlPort directly?

Reply:
I guess we could, but I think this is clearer because you don't have to reason about whether the port is currently open or not, and you just want the address anyway, not an actual `Port` object.

Reply:
Spoke with Tyler 1:1, and **tl;dr** doing what I am asking would be a much bigger refactor than I realized